packages/react-static-plugin-less/src/node.api.js

```diff
@@ -3,7 +3,7 @@ import autoprefixer from 'autoprefixer'
 import postcssFlexbugsFixes from 'postcss-flexbugs-fixes'
 import semver from 'semver'
 
-export default ({ includePaths = [], ...rest }) => ({
+export default ({ includePaths = [], cssLoaderOptions, ...rest }) => ({
```

Comment:
```suggestion
export default ({ includePaths = [], cssLoaderOptions = {}, ...rest }) => ({
```

Probably want to default to an empty object?